Formula/cproto.rb

```diff
@@ -10,6 +10,11 @@ class Cproto < Formula
     "GPL-3.0-or-later" => { with: "Autoconf-exception-3.0" },
   ]
 
+  livecheck do
+    url "https://invisible-mirror.net/archives/cproto/"
+    regex(/href=.*?cproto[._-]v?(\d+(?:[.-]\d+)+[a-z])\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?cproto[._-]v?(\d+(?:\.\d+)+[a-z]?)\.t/i)
```

It doesn't seem to be necessary to use `[.-]` (instead of the usual `\.`) in this particular context. We usually only modify this part of the regex when we know it's necessary to be able to properly match versions.

It may also be a good idea to make the trailing letter in the version optional, in case it's ever omitted (accidentally or intentionally).